src/packages/frontend/project/explorer/path-navigator.tsx

```diff
@@ -23,44 +23,46 @@ export const PathNavigator: React.FC<Props> = React.memo((props: Props) => {
 
   function make_path(): JSX.Element[] {
     const v: JSX.Element[] = [];
-    v.push(
-      // yes, must be called as a normal function
-      PathSegmentLink({
-        path: "",
-        display: <HomeOutlined />,
-        full_name: "",
-        key: 0,
-        on_click: () => actions?.open_directory("", true, false),
-      })
-    );
-
-    const is_root = current_path[0] === "/";
 
     const current_path_depth =
       (current_path == "" ? 0 : current_path.split("/").length) - 1;
     const history_segments = history_path.split("/");
+    const is_root = current_path[0] === "/";
+
+    v.push(
+      // yes, must be called as a normal function
```

Comment:
Ooops  -- don't leave a comment in "yes, must be called as a normal function" when the code *doesn't* do that anymore.